### PR TITLE
25162 fixed wrong format link

### LINKED ISSUE #25162: Message at Frontend has No HTML format
### Preconditions (*)

1. Magento 2.3-develop

### Steps to reproduce (*)

1. Go to backend
2. Store-> configuration-> Customer -> Newsletter
3. 
image
"Allow Guest Subscription" is "No"
4. Go to frontend
5. Enter the email "test@localhost.com" , press "Subscribe" at the bottom.
6. Look the message:
image

### Expected result (*)

image

### Actual result (*)

image

### PR DESCRIPTION
**Pull request by Usik2203**
### Description (*)

This PR fixed #25162 issue
72064499-ab887300-32e4-11ea-82de-2408bfd26459

### Fixed Issues (if relevant)

1. magento/magento2#25162: Message at Frontend has No HTML format

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento give me test instance

**Comment by Usik2203** [comment]
@magento give me test instance

**Comment by Usik2203** [comment]
Hi @rogyar .
I have closed wrong PR https://github.com/magento/magento2/pull/26328  and created this.
I added MFTF tests as You mentioned before. 
I have some trouble with Adobe CLA but I will solve it soon.

Thank You.

**Comment by magento-engcom-team** [comment]
Hi @Usik2203. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Usik2203. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @Usik2203, here is your [new Magento instance](https://pr-26455.instances.magento-community.engineering).
Admin access: [https://pr-26455.instances.magento-community.engineering/admin_e8d7](https://pr-26455.instances.magento-community.engineering/admin_e8d7)
Login: `d1d1f60c` Password: `d41451532ea1`
Instance will be terminated in up to 3 hours.

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup">
```
Comment (lbajsarowicz) [inline app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup.xml]: I'm not sure, but there's no assertion in this Action Group. Maybe there's other name that is better fit?

File: app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<pages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/PageObject.xsd">
+    <page name="AdminNewsletterConfigPage" url="admin/system_config/edit/section/newsletter/" area="admin" module="Magento_Newsletter">
```
Comment (lbajsarowicz) [inline app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml:10]: You don't need to provide `admin/` at the beginning.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
Only small changes to be applied.

File: app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml (hunk 2 above)
Comment (Usik2203) [reply app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml:10]: Hi @lbajsarowicz. 
After changes  `url="admin/system_config/edit/section/newsletter/"` to `url="system_config/edit/section/newsletter/"` 

I am getting such issue. I can't visit config page correctly.
Screenshot 2020-01-27 at 13 38 03

Also I couldn't find examples in core without `admin/`
Could You Please advise me something about it. If you can.

Thanks.

Comment (lbajsarowicz) [reply app/code/Magento/Newsletter/Test/Mftf/Page/AdminNewsletterConfigPage.xml:10]: OK. I'm sorry, revert `admin/` then. I'll take a look into it later :-)

File: app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Newsletter/Test/Mftf/ActionGroup/AssertAdminNewsletterConfigFieldGuestNotAllowedActionGroup.xml]: Hi @lbajsarowicz .
I have applied Your recommendation. 

Thank you for your advice.

File: app/code/Magento/Newsletter/Test/Mftf/Test/VerifyRegistredLinkDisplayedForGuestSubscriptionNoTest.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="VerifyRegistredLinkDisplayedForGuestSubscriptionNoTest">
+        <before>
+            <!--Log in to Magento as admin.-->
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Newsletter/Test/Mftf/Test/VerifyRegistredLinkDisplayedForGuestSubscriptionNoTest.xml]: Use `LoginActionGroup` instead, please.

File: app/code/Magento/Newsletter/Test/Mftf/Test/VerifyRegistredLinkDisplayedForGuestSubscriptionNoTest.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+            <!--Allow Guest Subscription NO-->
+            <actionGroup ref="AdminNewsletterConfigFieldSetGuestNotAllowedActionGroup" stepKey="amOnNewsletterConfigField"/>
+            <!--Log out from Magento admin.-->
+            <actionGroup ref="logout" stepKey="logoutFromAdmin"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/Newsletter/Test/Mftf/Test/VerifyRegistredLinkDisplayedForGuestSubscriptionNoTest.xml]: Use `AdminLogoutActionGroup` please

**Review by lbajsarowicz — APPROVED** [review]
Just minor changes to be introduced. Great job! : )

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6691](https://jira.corp.magento.com/browse/ENGCOM-6691) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Review by lbajsarowicz — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6691](https://jira.corp.magento.com/browse/ENGCOM-6691) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Before:**
before

**After:**
after

**Comment by engcom-Golf** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
@magento run all tests

**Pull request merged**